### PR TITLE
RavenDB-14471 - Indexing processing speed doesn't include side by side indexes

### PR DESCRIPTION
**Pull request by grisha-kotler**
https://issues.hibernatingrhinos.com/issue/RavenDB-14471

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts [hunk 1]
```diff
@@ -110,8 +110,26 @@ class indexes extends viewModelBase {
...
+
+                        const replacement = i.replacement();
+                        if (replacement) {
+                            const replacementProgress = replacement.progress();
+                            if (replacementProgress) {
+                                sum += replacementProgress.globalProgress().processedPerSecond();
+                            }
+                        }
+
+                        console.log(sum);
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts]: remove?

Comment (grisha-kotler) [reply src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts]: done

File: test/FastTests/Client/CRUD.cs [hunk 2]
```diff
@@ -12,6 +12,24 @@ public CRUD(ITestOutputHelper output) : base(output)
         {
         }
 
+        public class TestDocument
```
Comment (ppekrol) [inline test/FastTests/Client/CRUD.cs]: What is this? Can we remove this?

Comment (grisha-kotler) [reply test/FastTests/Client/CRUD.cs]: done

**Review by ml054 — APPROVED** [review]

**Pull request merged**